static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -22,13 +22,6 @@ export function mapResponseToReplayRecord(apiResponse: any): ReplayRecord {
     ...apiResponse,
     ...(apiResponse.startedAt ? {startedAt: new Date(apiResponse.startedAt)} : {}),
     ...(apiResponse.finishedAt ? {finishedAt: new Date(apiResponse.finishedAt)} : {}),
-    user: {
-      email: apiResponse.user.email || '',
-      id: apiResponse.user.id || '',
-      ip_address: apiResponse.user.ip_address || '',
-      name: apiResponse.user.name || '',
-      username: '',
-    },
```

Comment:
This was a landmine. Removed it and made the type reflect what's in https://github.com/getsentry/replay-backend/blob/main/blueprints/api.md more closely.

Reply:
💣